paymentsheet/src/main/java/com/stripe/android/lpmfoundations/paymentmethod/definitions/LinkCardBrandDefinition.kt

```diff
@@ -0,0 +1,50 @@
+package com.stripe.android.lpmfoundations.paymentmethod.definitions
+
+import com.stripe.android.lpmfoundations.luxe.SupportedPaymentMethod
+import com.stripe.android.lpmfoundations.paymentmethod.AddPaymentMethodRequirement
+import com.stripe.android.lpmfoundations.paymentmethod.PaymentMethodDefinition
+import com.stripe.android.lpmfoundations.paymentmethod.PaymentMethodMetadata
+import com.stripe.android.lpmfoundations.paymentmethod.UiDefinitionFactory
+import com.stripe.android.model.PaymentMethod
+import com.stripe.android.uicore.elements.FormElement
+import com.stripe.android.ui.core.R as PaymentsUiCoreR
+
+internal object LinkCardBrandDefinition : PaymentMethodDefinition {
```

Comment:
This is identical to `InstantDebitsDefinition`. Once we move the bank form to a proper `UiDefinitionFactory`, we can also deduplicate this.